include/z64player.h

```diff
@@ -392,13 +392,13 @@ typedef enum PlayerFacialExpression {
     /* 15 */ PLAYER_FACE_15
 } PlayerFacialExpression;
 
-#define GET_EXPRESSION_FROM_JOINTTABLE(jointTable)        (((PlayerAnimationFrame *)(jointTable))->expressionInfo)
-#define GET_EYE_INDEX_FROM_JOINTTABLE(jointTable)         ((GET_EXPRESSION_FROM_JOINTTABLE(jointTable) & 0xF) - 1)
-#define GET_MOUTH_INDEX_FROM_JOINTTABLE(jointTable)       (((GET_EXPRESSION_FROM_JOINTTABLE(jointTable) >> 4) & 0xF) - 1)
+#define GET_GESTURE_FROM_JOINTTABLE(jointTable)           (((PlayerAnimationFrame *)(jointTable))->expressionInfo)
```

Comment:
Could change `expressionInfo` -> `gestureInfo`, and it's probably worth a comment at that struct mentioning it contains info on eyes, mouth, and hands (or face and hands)